packages/test/src/test-testenvironment.ts

```diff
@@ -0,0 +1,40 @@
+import anyTest, { TestInterface } from 'ava';
+import { v4 as uuid4 } from 'uuid';
+import { TestWorkflowEnvironment } from '@temporalio/testing';
+import { Worker } from '@temporalio/worker';
+import { defaultOptions } from './mock-native-worker';
+import { sleeper } from './workflows';
+
+interface Context {
+  testEnv: TestWorkflowEnvironment;
+}
+
+const test = anyTest as TestInterface<Context>;
+
+test.before(async (t) => {
+  t.context = {
+    testEnv: await TestWorkflowEnvironment.create(),
+  };
+});
+
+test.after.always(async (t) => {
+  await t.context.testEnv?.teardown();
+});
+
+test('TestEnvironment sets up test server and is able to run a Workflow with time skipping', async (t) => {
+  const worker = await Worker.create(defaultOptions);
```

Comment:
Would be good to have more than one test in here to make sure things work properly then. We need to require they are done serially?

Reply:
Tests would probably need to be run serially.
Right now the test server is still missing some critical features, I'll add a bunch more tests when those features are availabe.